pkgs/by-name/pr/privatebin/package.nix

```diff
@@ -22,13 +24,19 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
-  passthru.tests = nixosTests.privatebin;
+  passthru = {
+    tests = nixosTests.privatebin;
+    updateScript = nix-update-script { };
+  };
 
   meta = {
     changelog = "https://github.com/PrivateBin/PrivateBin/releases/tag/${finalAttrs.version}";
     description = "Minimalist, open source online pastebin where the server has zero knowledge of pasted data.";
     homepage = "https://privatebin.info";
-    license = lib.licenses.gpl2;
-    maintainers = [ lib.maintainers.savyajha ];
+    license = lib.licenses.gpl2Only;
```

Comment:
The license of `privatebin` seems to be `zlib`. Additionally I think GPLv2, MIT, BSD3, Apache 2 and CC-BY should be included because the dependencies are directly included in the repo and this package.
https://github.com/PrivateBin/PrivateBin/blob/master/LICENSE.md